lib/browser.rb

```diff
@@ -1,15 +1,22 @@
 class Browser
   def initialize(options = {})
     @found_jobs = []
-    @keywords = options[:keywords] || "ruby, rails"
+    @keywords = options[:keywords] || "product"
     @link_parser = LinkParser.new
     @url_validator = UrlValidator.new
+
+    @num_pages = ARGV[0] ? ARGV[0].to_i : 1
   end
 
   def scour_aggregator(aggregator)
     @aggregator = aggregator
-    page = browse_to_url(@aggregator.url)
-    scour_page(page)
+
+    (1..@num_pages).each do |page_id|
+      @aggregator.url = "https://nytm.org/made?list=true&page=#{page_id}"
```

Comment:
Definitely see what you'er saying. That was my main concern, just couldn't put it into words.

So I think what I'll attempt is to check for ARGV in ```JobAggregator```, then set num_pages to > 1 if it exists.

That way in #scour_aggregator I can iterate over the pages by just looking for the number of pages and tacking on the URL param.